test/unit/queue/src/QueueTest.cpp

```diff
@@ -159,7 +161,27 @@ void operator()()
 }
 };
 
-using TestQueues = alpaka::test::queue::TestQueues;
+//#############################################################################
+//! A std::tuple holding tuples of devices and corresponding queue types.
+using TestQueues =
+    std::tuple<
+        std::tuple<alpaka::dev::DevCpu, alpaka::queue::QueueCpuBlocking>,
+        std::tuple<alpaka::dev::DevCpu, alpaka::queue::QueueCpuNonBlocking>
+#ifdef ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED
+        ,
+        std::tuple<alpaka::dev::DevCpu, alpaka::queue::QueueCpuOmp2Collective>
+#endif
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+        ,
+        std::tuple<alpaka::dev::DevCudaRt, alpaka::queue::QueueCudaRtBlocking>,
+        std::tuple<alpaka::dev::DevCudaRt, alpaka::queue::QueueCudaRtNonBlocking>
+#endif
+#ifdef ALPAKA_ACC_GPU_HIP_ENABLED
+        ,
+        std::tuple<alpaka::dev::DevHipRt, alpaka::queue::QueueHipRtBlocking>,
+        std::tuple<alpaka::dev::DevHipRt, alpaka::queue::QueueHipRtNonBlocking>
+#endif
+    >;
```

Comment:
Why did you remove it completely from `test/queue/Queue.hpp`? I would prefer to still have the `TestQueues` there and only appending the `QueueCpuOmp2Collective` here.

Reply:
Or simply add it to `TestQueues` there.

Reply:
I moved it to the tests because in this file defines traits like `IsBlockingQueue`. This make it impossible to have all trait definitions outside of the file `Queue.hpp` but I do not like to define the traits within `Queue.hpp` because I like to have clear borders between the "user code` (the collective queue) and alpaka.

Reply:
But then please leave the alpaka-queues in there and append the new one.